src/mudlet.cpp

```diff
@@ -3257,6 +3257,12 @@ void mudlet::readEarlySettings(const QSettings& settings)
     }
 
     mInterfaceLanguage = settings.value("interfaceLanguage", autodetectPreferredLanguage()).toString();
+    mUserLocale = QLocale(mInterfaceLanguage);
+    if (mUserLocale == QLocale::c()) {
+        qWarning().nospace().noquote() << "mudlet::readEarlySettings(...) WARNING - Unable to convert language code \"" << mInterfaceLanguage << "\" to a recognised locale, reverting to the POSIX 'C' one...";
```

Comment:
The ellipsis isn't necessary in any of the statements here.

Reply:
Not sure I understand the suggested change

Reply:
At the end of the message, the `...` is redundant (or incorrect). `...` is used if there is an action that is going to follow.

Reply:
Well you know how I am with the use of ellipses... :wink: 

... in this case I suppose I was sort of saying "Well that is what has happened, if it breaks then you get to keep the pieces!" But I can do an in-place edit here to trim them out.